src/commands/mobile_app/upload.rs

```diff
@@ -67,12 +102,21 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         .get_many::<String>("paths")
         .expect("paths argument is required");
 
-    let sha = matches
-        .get_one("sha")
+    let head_sha = matches
+        .get_one("head_sha")
         .map(String::as_str)
         .map(Cow::Borrowed)
         .or_else(|| vcs::find_head().ok().map(Cow::Owned));
 
+    // TODO: Implement default values
```

Comment:
Will follow in future PRs for each of these individually.

Reply:
I am okay with us doing this in future PRs; however, I would prefer not to commit the `TODO` comment, lest we ever forget to remove it. We should track the work in GitHub/Linear issues instead

```suggestion
```